app/services/spree/product_feed_service.rb

```diff
@@ -91,8 +95,8 @@ def availability
     end
 
     def image_link
-      return unless product.try(:images).any?
-      "#{store.url}#{product.try(:images).first.try(:attachment).url(:large)}"
+      return "#{store.url}#{variant.try(:images).first.try(:attachment).try(:url, :product_feed)}" if variant.images.any?
+      return "#{store.url}#{product.try(:images).first.try(:attachment).try(:url, :product_feed)}" if product.images.any?
```

Comment:
What if paperclip is setup to use a CDN? I'm pretty sure that `attachment.url` returns the full image url if paperclip configs use a custom interpolator to use a CDN

Reply:
yes, but here we need it as `product_feed` format, the `attachment.url` method should return the default style and that's `product`